service/controller/app/app.go

```diff
@@ -11,7 +11,7 @@ import (
 	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	"k8s.io/client-go/kubernetes"
 
-	"github.com/giantswarm/app-operator/service/controller/app/v1"
+	v1 "github.com/giantswarm/app-operator/service/controller/app/v1"
```

Comment:
I would say either `appv1` or no alias. This one is kind of pointless.

Reply:
Good catch. This was just my editor misbehaving. Fixed.